l5kit/l5kit/tests/data/zarr_utils_test.py

```diff
@@ -135,3 +136,66 @@ def test_zarr_split(dmg: LocalDataManager, tmp_path: Path, zarr_dataset: Chunked
             assert np.all(input_frames["ego_rotation"] == output_frames["ego_rotation"])
             assert np.all(input_agents == output_agents)
             assert np.all(input_tl_faces == output_tl_faces)
+
+
+@pytest.mark.parametrize("num_frames_to_copy", [1, 10, 50, pytest.param(500, marks=pytest.mark.xfail)])
+def test_zarr_scenes_chunk(
+    dmg: LocalDataManager, tmp_path: Path, zarr_dataset: ChunkedDataset, num_frames_to_copy: int
+) -> None:
+    # first let's concat so we have multiple scenes
+    concat_count = 10
+    zarr_input_path = dmg.require("single_scene.zarr")
+    zarr_cat_path = str(tmp_path / f"{uuid4()}.zarr")
```

Comment:
what is `cat`? xD

Reply:
concatenate xD

Reply:
I would change `cat` -> `concat`

Reply:
can't do that, it's the name of the function lol

Reply:
well, then `zarr_concated`

Reply:
what about changing the function name to `zarr_concatenate` ?

Reply:
@lucabergamini I would keep the function name to `zarr_concat`, I thought you like short name :)

Reply:
done it :)